integration-tests/README.markdown

```diff
@@ -0,0 +1,24 @@
+# Integration Tests
+
+__The integration tests need explicit activation by setting the ``skip.integration.tests``
+tunable to *false*.__
+All tests require a running Graylog server to run against, which is not bootstrapped by maven but supposedly
+by the environment the integration tests are running on. Therefore, the tests have a number of tunables with (hopefully)
+reasonable defaults. These are:
+
+| Property               | Default | Description |
+|:-----------------------|:--------|:------------|
+| skip.integration.tests | false   | Enables the integration tests. (Disabled per default) |
+| gl2.port | ``12900`` | Tcp port of the graylog server. |
+| gl2.baseuri | ``"http://localhost"`` | Uri base of the graylog server. Port is overriden by gl2.port if specified. |
+| gl2.admin_user | ``admin`` | Username used for authenticating against server for authenticated requests. |
+| gl2.admin_password | ``admin`` | Password used for authenticating against server for authenticated requests. |
+| mongodb.host | ``"localhost"`` | Hostname of MongoDB server used for seeding tests that require a preseeded database. |
+| mongodb.port | ``27017`` | Tcp port of MongoDB server. |
+| mongodb.database | ``"graylog"`` | Default database name for MongoDB. Tests are able to override this. |
```

Comment:
I think the default should be something like "graylog_test" or "graylog_it" to prevent accidentally overwriting and deleting the locally running MongoDB database on developer setups.